### ISSUE TITLE
Bad log formatting in regression test

### ISSUE THREAD

**Issue by GalRogozinski** [iotaledger/iri#1607]
### Bug description
Aloe outputs a non-formatted log message upon error. Thus obscuring it..

See 
https://buildkite.com/iota-foundation/iri-regression-tests/builds/1035#64546483-11d4-49dc-bd61-079309c1fd54/30-155

### IRI version
1.8.1

### Steps To Reproduce
1. Run regression tests on https://github.com/iotaledger/iri/pull/1476/commits/3371aaccf35799151f29b1aa71ef8bc30fb33671
2.Look at machine 6 tests

### Expected behaviour
Log should be formatted...

### Actual behaviour
```
assert 'ixi' in request_return, "Error: {}".format(request_return['error'])
--
  | KeyError: 'error'
```

**Comment by GalRogozinski** [comment]
Recommended formatting could be done like so:
`formatter = logging.Formatter(fmt='%(asctime)s - %(levelname)s - %(module)s - %(message)s')`

Perhaps it is a good idea to create a custom logger factory so we can create loggers that have different modules but similar formatting (and perhaps other similar configs).

Motivation for timestamps is that we can relate it to similar events on the node log

**Comment by GalRogozinski** [comment]
Fixed by #1743
